app/admin/members.rb

```diff
@@ -29,7 +29,7 @@
       group_links = member.groups.map { |group| auto_link group }
       safe_join group_links, ', '
     end
-    column :worked_hours_of_three_last_months
+    column(t('.worked_hours')) { |member| MemberDecorator.new(member).hours_worked_in_the_three_last_months }
```

Comment:
You shouldn't need to use `MemberDecorator.new(member)` here, as you have already declared the use of the `MemberDecorator` in this ActiveAdmin resource (line 21)
Does it work with just `{ |member| member.hours_worked_in_the_last_three_months }` ?

Reply:
done